README.md

```diff
@@ -2,14 +2,10 @@
 
  <img align="right" src="https://github.com/tauri-apps/tauri/raw/dev/app-icon.png" height="128" width="128">
 
-[![status](https://img.shields.io/badge/Status-Beta-green.svg)](https://github.com/tauri-apps/tauri)
-[![Chat Server](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/SpmNs4S)
-[![devto](https://img.shields.io/badge/blog-dev.to-black.svg)](https://dev.to/tauri)
-
-![](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library
-)
+[![status](https://img.shields.io/badge/Status-Stable-green.svg)](https://github.com/tauri-apps/tauri)
+[![Chat Server](https://img.shields.io/badge/chat-discord-7289da.svg)](https://discord.gg/SpmNs4S)
+![](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)
```

Comment:
is this label supposed to be something else other than `test-library`?

Reply:
should we delete the badge or change the label to just `tests`?

Reply:
Also I think this badge exists in the main tauri repo

Reply:
I'm not sure for the one on line 7, I just moved it up a line but it was already there. I'm assuming our testing will need to be redone after the re-write so maybe we keep as-is and update it when we update tests?

Reply:
Oh, just realised it's pointing to the main repo, let me just remove that label from here